client/app/components/routes/profile/User.tsx

```diff
@@ -28,7 +28,13 @@ export default async function User() {
        * getServerSession requires passing the same object you would pass to NextAuth
        */
       const session = await getServerSession(authOptions);
-      const names = session?.user?.name?.split(" ");
+      const isIdir = session?.identity_provider === "idir";
```

Comment:
We have some enums to help with this check: /bc_obps/registration/enums/enums.py, e.g. of use: /bc_obps/registration/api/user.py

Reply:
Sounds good, though I don't think we should be importing anything from the backend even if we are a monorepo. I created an `IDP` type with the other types like we do for `Status` in `client/app/types/types.ts`.